### PR TITLE
Stage new files

### PR DESCRIPTION
**Pull request by ari-gold**

### CODE REVIEW

File: source/_docs/drupal-security-patches.md [hunk 1]
```diff
@@ -88,6 +89,7 @@ You'll need a local git clone of your site code.
 
     ```bash
     rm 119.patch
+    git add includes/request-sanitizer.inc
```
Comment (greg-1-anderson) [inline source/_docs/drupal-security-patches.md]: This is incomplete -- there are more files that need to be added. Can we just say `git add .` here?

Comment (ari-gold) [reply source/_docs/drupal-security-patches.md]: the next line is `git commit -am`

File: source/_docs/drupal-security-patches.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ You'll need a local git clone of your site code.
 
     ```bash
     rm 206.patch
+    git add core/lib/Drupal/Core/Security/
```
Comment (greg-1-anderson) [inline source/_docs/drupal-security-patches.md]: There are files in other locations that also need to be added. Can we just say `git add .` instead?

**Comment by greg-1-anderson** [comment]
+1 LGTM.

Looks like the test failure was transient, perhaps caused by high load from folks updating.

**Pull request merged**